src/PresentationalComponents/PlainTextWithLink.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, TextVariants } from '@patternfly/react-core';
+
+export const PlainTextWithLink = ({ label, linkText, linkUrl }) => {
+    return (
+        <Text component={ TextVariants.p }>
+            {label}
+            &nbsp;&nbsp;
```

Comment:
You can put a left marging to the link. i.e. `pf-u-ml-md`

Reply:
Cool! Thanks